docs/lib/snippets/dart_api/manager.dart

```diff
@@ -332,3 +332,65 @@ Future<void> customOrdering(AppDatabase db) async {
   db.managers.todoItems.orderBy((f) => f.contentThenCreatedAt());
 }
 // #enddocregion manager_custom_ordering
+
+void _managerAnnotations(AppDatabase db) async {
+// #docregion manager_annotations
+// First create an annotation with an expression you want to use
```

Comment:
Oh it is the snippet haha - check the website, the docregion comment doesn't have to be indented but IMO the comments should be aligned with the line they're annotating (easier to read and makes the snippet ready to copy).